client/src/component/Title/Title.css

```
@@ -0,0 +1,98 @@
+body {
+background-image: url("Background.png");
+background-repeat: no-repeat;
+    min-height: 92vh;
+    display: flex;
+    flex-direction: column;
+background-attachment: fixed;
+    text-align: center;
+    color: white;
+    font-size: 35px !important;
+    margin: 10px;
+}
+
+.box {
+    width: 40%;
+    margin: 0 auto;
+    background: rgba(255,255,255,0.2);
+    padding: 35px;
+    border: 2px solid #fff;
+    border-radius: 20px/50px;
+    background-clip: padding-box;
+    text-align: center;
+  }
+  
+  .button {
+    font-size: 1em;
+    padding: 0px;
+    color: #fff;
+    border: 2px solid #354e75;
+    border-radius: 20px/50px;
+    text-decoration: none;
+    cursor: pointer;
+    transition: all 0.3s ease-out;
+  }
+  .button:hover {
+    background: #354e75;
+  }
+  
+  .overlay {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: rgba(0, 0, 0, 0.7);
+    transition: opacity 500ms;
+    visibility: hidden;
+    opacity: 0;
+  }
```

```suggestion
  }
  
```

space after the `.overlay` and before the `.overlay:target`